<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>778</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="st_tab">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="dest_tab">
       <attribute name="title">
        <string>Destinations</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="check_cities">
            <property name="text">
             <string>Check destinations</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="del_city">
            <property name="text">
             <string>Delete destination</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="new_city">
            <property name="text">
             <string>New destination</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="cities_table">
          <column>
           <property name="text">
            <string>Country</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Destination</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Available</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Students</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string/>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Find</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="searchthis">
             <property name="text">
              <string notr="true">*</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>in</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="searchhere"/>
           </item>
           <item>
            <widget class="QCheckBox" name="searchfromstart">
             <property name="text">
              <string>Start with first student</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="searchst">
             <property name="text">
              <string>Search student</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="prev_student">
            <property name="text">
             <string>Previous</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="next_student">
            <property name="text">
             <string>Next</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="4">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Surname:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLineEdit" name="frm_name"/>
          </item>
          <item row="2" column="3">
           <widget class="QLineEdit" name="frm_dest2"/>
          </item>
          <item row="0" column="5">
           <widget class="QLineEdit" name="frm_surname"/>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Destination2:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <widget class="QLineEdit" name="frm_meanvote"/>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="frm_dest1"/>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="frm_ID"/>
          </item>
          <item row="1" column="4">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Average mark:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Destination1:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Curricula:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Current year:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="frm_curricola"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>ID:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Name:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="frm_dest3"/>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Destination4:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Destination3:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QLineEdit" name="frm_dest4"/>
          </item>
          <item row="2" column="5">
           <widget class="QCheckBox" name="frm_requisites">
            <property name="text">
             <string>Requisites</string>
            </property>
           </widget>
          </item>
          <item row="3" column="5">
           <widget class="QCheckBox" name="frm_former">
            <property name="text">
             <string>Former Erasmus</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QComboBox" name="frm_registration">
            <property name="currentText">
             <string notr="true">1</string>
            </property>
            <item>
             <property name="text">
              <string>1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>3</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>5</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="new_student">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(170, 255, 127);</string>
            </property>
            <property name="text">
             <string>Save as new student</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="del_student">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 117, 117);</string>
            </property>
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="save_student">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 127);</string>
            </property>
            <property name="text">
             <string>Modify student</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="students_table">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <column>
           <property name="text">
            <string>ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Surname</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Curricula</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Current yr</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Average mark</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Requisites</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Former Erasmus</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Destination1</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Destination2</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Destination3</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Destination4</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Ranking</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="easteregg">
           <property name="styleSheet">
            <string notr="true">Text-align:left</string>
           </property>
           <property name="text">
            <string>Calculate ranking</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="calculateRanking">
           <property name="text">
            <string>Calculate ranking</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="ranking_table">
         <column>
          <property name="text">
           <string>ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Surname</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Curricula</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Ranking</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Auto chosen</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Manually chosen</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>727</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen_project"/>
    <addaction name="actionSave_project"/>
    <addaction name="actionSave_project_as"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuProject">
    <property name="title">
     <string>Pro&amp;ject</string>
    </property>
    <addaction name="actionImport_xls"/>
    <addaction name="actionExport_ranking_xslx"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Abo&amp;ut</string>
    </property>
    <addaction name="actionAbout_Cresta"/>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>L&amp;anguage</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuProject"/>
   <addaction name="menuLanguage"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>&amp;New project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen_project">
   <property name="text">
    <string>&amp;Open project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_project">
   <property name="text">
    <string>&amp;Save project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_project_as">
   <property name="text">
    <string>Save &amp;project as</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionImport_xls">
   <property name="text">
    <string>&amp;Import students xlsx</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionAbout_Cresta">
   <property name="text">
    <string>&amp;About Cresta</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>&amp;Help</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="actionExport_ranking_xslx">
   <property name="text">
    <string>&amp;Export ranking xslx</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>frm_ID</tabstop>
  <tabstop>frm_name</tabstop>
  <tabstop>frm_surname</tabstop>
  <tabstop>frm_curricola</tabstop>
  <tabstop>frm_meanvote</tabstop>
  <tabstop>frm_dest1</tabstop>
  <tabstop>frm_dest2</tabstop>
  <tabstop>frm_dest3</tabstop>
  <tabstop>frm_dest4</tabstop>
  <tabstop>frm_requisites</tabstop>
  <tabstop>frm_former</tabstop>
  <tabstop>next_student</tabstop>
  <tabstop>st_tab</tabstop>
  <tabstop>ranking_table</tabstop>
  <tabstop>prev_student</tabstop>
  <tabstop>students_table</tabstop>
  <tabstop>cities_table</tabstop>
  <tabstop>calculateRanking</tabstop>
  <tabstop>new_city</tabstop>
  <tabstop>del_city</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
